SUPPORT.md

```diff
@@ -2,10 +2,10 @@
 
 Here are some useful sources of information about using Mbed TLS:
 
+- [ReadTheDocs](https://mbed-tls.readthedocs.io/);
 - API documentation, see the [Documentation section of the
   README](README.md#License);
 - the `docs` directory in the source tree;
-- the [Mbed TLS knowledge Base](https://tls.mbed.org/kb);
```

Comment:
why remove this rather than change the link to `https://mbed-tls.readthedocs.io/en/latest/kb/`?

Reply:
It's a subset of the ReadTheDocs link added at the top - seems unnecessarily redundant?

Reply:
It seems like a useful subset to specifically point out (which we do in a number of places)

Reply:
OK, restored